src/components/categories/categories.css

```diff
@@ -0,0 +1,29 @@
+.left-bar {
+  position: absolute;
+  left: auto;
+  width: 50px;
+  height: 10px;
+  line-height: 40px;
+  padding-left: 5%;
+  color: #0070f3;
+  font-weight: bolder;
+  font-size: 17px;
+  text-transform: uppercase;
+  text-align: left;
+}
+.left-bar button {
+  text-align: center;
+  white-space: nowrap;
+}
+.left-bar button:hover {
+  background-color: #d0d8e2;
+}
+.left-bar button:focus {
+  outline: 1px dashed #d0d8e2;
+}
```

Comment:
Here on focus, can you either make it bold or something else which is more visible ?

Reply:
what do you think of this focus?

![Screenshot (8)](https://user-images.githubusercontent.com/20721985/103413924-4a585000-4b84-11eb-8825-00d9a63518c7.png)